src/crypto/crypto.gni

```diff
@@ -13,14 +13,10 @@
 # limitations under the License.
 
 declare_args() {
-  # Crypto implementation: mbedtls, openssl, boringssl, platform.
+  # Crypto implementation: mbedtls, psa, openssl, boringssl, platform.
   chip_crypto = ""
   chip_with_se05x = 0
 
-  # Compile mbedtls externally. Only used if chip_crypto == "mbedtls"
+  # Compile mbedtls externally.
   chip_external_mbedtls = false
 }
-
-assert(
```

Comment:
Does `chip_external_mbedtls` make sense for other crypto backend than mbedtls or psa?